src/stdio-bridge/stdio-bridge.c

```diff
@@ -131,7 +158,11 @@ int main(int argc, char *argv[]) {
                 goto finish;
         }
 
-        r = sd_bus_set_address(a, arg_bus_path);
+        if (arg_transport == BUS_TRANSPORT_MACHINE)
+                r = bus_set_address_system_machine(a, arg_bus_path);
+        else
+                r = sd_bus_set_address(a, arg_bus_path);
```

Comment:
I prefer here the following: `arg_bus_path ?: DEFAULT_BUS_PATH`. Then we can avoid strdup()ing the constant.